### ISSUE TITLE
Find → “Ignore Case” doesn’t work in CotEditor 2.4.0-beta (97)

### ISSUE THREAD

**Issue by handcoding** [coteditor/CotEditor#503]
In CotEditor 2.4.0-beta (97), it would appear that “Ignore Case” doesn’t work within the Find dialog. (That is, the Find dialog appears to be case sensitive whether “Ignore Case” is checked or not.)

Here’s a test case:

1. Paste this into an empty document in CotEditor:
    ```
    * What the "Show Style Guides” and “Show Direct Formatting Guides” options do within Word:
	http://savethesemicolon.com/2011/04/16/formatting-guides-in-word-for-mac-2011/

    * A bunch of handy Word shortcuts:
	http://jeromyanglim.tumblr.com/post/37897099625/list-of-important-hidden-shortcut-keys-for
    
    * Adrienne Montgomerie’s recommended settings for Word:
	http://www.copyediting.com/how-get-word-ready-editing
    ```

2. Hit Cmd+F to open the Find dialog:
    * Make sure that “Ignore Case” is checked
    * Search for the string “word” (without the quote marks).

3. Result: CotEditor is able to find the instances of “word” that happen to be in lowercase, but it skips over instances of “Word” that are capitalized.

**Comment by 1024jp** [comment]
Oh! Thanks for the report. I didn't notice that. Shame.

However unfortunately, the v2.4.0 has already just been released. I'll fix it on the 2.4.1.
Please wait for the next release. It must be done in this month.

**Comment by handcoding** [comment]
Thanks for taking care of this! ⭐️
